### ISSUE TITLE
[Crash Report]: App crashes when shuffle a empty playlist

### ISSUE THREAD

**Issue by ssssy1000** [chr56/Phonograph_Plus#95]
### Crash log

```
Crash Report:

Package name:    player.phonograph.plus.checkout.debug
App version:     0.7.1-dev1 (601)
Git Commit Hash: affb407c5e5d449034c7
Android version: 7.1.1 (SDK 25)
Device brand:    google  (by Google)
Device model:    Android SDK built for x86 (code: generic_x86)
Product name:    sdk_google_phone_x86
Build version:   sdk_google_phone_x86-userdebug 7.1.1 NYC 6695155 test-keys 
                 (6695155)
Hardware:        ranchu
Language:        en

java.lang.IllegalArgumentException: n must be positive
	at java.util.Random.nextInt(Random.java:391)
	at kotlin.random.AbstractPlatformRandom.nextInt(PlatformRandom.kt:42)
	at kotlin.random.Random$Default.nextInt(Random.kt:283)
	at player.phonograph.actions.PlaylistActionsKt.actionShuffleAndPlay(PlaylistActions.kt:29)
	at player.phonograph.actions.menu.PlaylistKt$playlistToolbar$1$1$2$1.invoke(Playlist.kt:61)
	at player.phonograph.actions.menu.PlaylistKt$playlistToolbar$1$1$2$1.invoke(Playlist.kt:61)
	at player.phonograph.actions.menu.PlaylistKt$playlistToolbar$1$1$2$inlined$sam$i$android_view_MenuItem_OnMenuItemClickListener$0.onMenuItemClick(MenuItemContext.kt)
	at androidx.appcompat.view.menu.MenuItemImpl.invoke(MenuItemImpl.java:155)
	at androidx.appcompat.view.menu.MenuBuilder.performItemAction(MenuBuilder.java:987)
	at androidx.appcompat.view.menu.MenuBuilder.performItemAction(MenuBuilder.java:977)
	at androidx.appcompat.widget.ActionMenuView.invokeItem(ActionMenuView.java:625)
	at androidx.appcompat.view.menu.ActionMenuItemView.onClick(ActionMenuItemView.java:156)
	at android.view.View.performClick(View.java:5637)
	at android.view.View$PerformClick.run(View.java:22429)
	at android.os.Handler.handleCallback(Handler.java:751)
	at android.os.Handler.dispatchMessage(Handler.java:95)
	at android.os.Looper.loop(Looper.java:154)
	at android.app.ActivityThread.main(ActivityThread.java:6119)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:886)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:776)


```

### Steps to reproduce the bug

1. create a new playlist
2. click the newly created playlist
3. click Shuffle playlist

### Screenshots or Screen recordings

<img width="189" alt="1688886444896" src="https://github.com/chr56/Phonograph_Plus/assets/98003627/aa6007fd-3b87-48d5-bde4-9b5fc9ca7ce3">
<img width="187" alt="1688886463756" src="https://github.com/chr56/Phonograph_Plus/assets/98003627/e266653a-e5bf-4099-b569-c543fde11e29">
<img width="182" alt="1688886502439" src="https://github.com/chr56/Phonograph_Plus/assets/98003627/10c7c7c3-db8c-45f0-ab39-66c2e55fa3dd">

### Version

0.7.1-dev1

**Comment by chr56** [comment]
Comfirmed!

**Comment by chr56** [comment]
Fixed in revision 437eaa3125b9fd66c9b1b7689e081c05fb5f3bd4.

**Comment by chr56** [comment]
[v0.7.2](https://github.com/chr56/Phonograph_Plus/releases/tag/v0.7.2) has released! close!
